### PR TITLE
Fix doc for Fedora 32 workaround

### PR DESCRIPTION
**Pull request by rcarrillocruz**

### CODE REVIEW

File: site/content/docs/user/known-issues.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ You can work around this by changing the `FirewallBackend` in the `/etc/firewall
 
 ```console
 sed -i /etc/firewalld/firewalld.conf 's/FirewallBackend=.*/FirewallBackend=iptables/'
-systemctl restart docker
+systemctl restart firewalld
```
Comment (aojea) [inline site/content/docs/user/known-issues.md:296]: Line above says `restarting docker` , seems the user submitted it found that restarting docker solved the problem.
Is ok restarting only firewalld?

Comment (aojea) [reply site/content/docs/user/known-issues.md:296]: please change line 292 too :)

**Comment by BenTheElder** [comment]
Thanks for the PR!

Unfortunately Kubernetes projects do require the CNCF CLA, we'll need that before we can merge this, in addition to review comments. https://github.com/kubernetes-sigs/kind/pull/1739#issuecomment-660892590

File: site/content/docs/user/known-issues.md (hunk 1 above)
Comment (rcarrillocruz) [reply site/content/docs/user/known-issues.md:296]: > Line above says `restarting docker` , seems the user submitted it found that restarting docker solved the problem.
> Is ok restarting only firewalld?

yep, I just needed to restart firewalld

**Comment by BenTheElder** [comment]
/approve
that 1.16 thing still needs a bug, sigh.

**Comment by BenTheElder** [comment]
filed https://github.com/kubernetes/kubernetes/issues/93301

**Pull request merged**